### DISCUSSION TITLE
Stuck on step 4

### DISCUSSION THREAD

**Discussion by zgall111** [riffnshred/nhl-led-scoreboard#369] [General]
I’m new to all of this and I’m definitely struggling. When I try to clone the github repo using `git clone --recursive https://github.com/riffnshred/nhl-led-scoreboard`, it asks me for my github username and password. When I enter my username and personal access token, I get “Authentication failed” error. Am I missing something?

**Comment by riffnshred** [comment]
Let me test this tonight, you should not have to authenticate.

**Comment by zgall111** [comment]
I was just able to get in after setting up an SSH key and using that.

**Comment by riffnshred** [comment]
Thats really strange, I just tested it and all works on my end. and I dont have any SSH key setup on my account
